### PR TITLE
Added EventLog

### PR DESCRIPTION
**Pull request by jhkimnew**
Added EventLog to support Windows Event log.

### CODE REVIEW

File: src/Microsoft.IIS.Administration/Program.cs [hunk 1]
```diff
@@ -29,13 +30,18 @@ public class Program {
...
                     // Console log is not available in running as a Service
                     if (!runAsAService) {
-                        logging.AddConfiguration(hostingContext.Configuration.GetSection("Logging"));
                         logging.AddConsole();
-                        logging.AddDebug();
-                    }
+                    }
+
+                    logging.AddDebug();
+                    logging.AddEventLog(new EventLogSettings() {
```
Comment (shhsu) [inline src/Microsoft.IIS.Administration/Program.cs]: Let's unify the event source with this
https://github.com/microsoft/IIS.Administration/blob/91ad2f8c4e206230f58fdf4e2a173bc2971e948b/src/Microsoft.IIS.Administration/Errors/ErrorHandler.cs#L43

Maybe poll out a constant

Comment (jhkimnew) [reply src/Microsoft.IIS.Administration/Program.cs]: Done

Comment (shhsu) [reply src/Microsoft.IIS.Administration/Program.cs]: We should make it a constant though

Comment (jhkimnew) [reply src/Microsoft.IIS.Administration/Program.cs]: Done.

**Pull request merged**